src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -74,6 +75,7 @@ public class DurableTaskExtension :
         private IDurabilityProviderFactory durabilityProviderFactory;
         private INameResolver nameResolver;
         private ILoggerFactory loggerFactory;
+        private ILogger logger;

```

Comment:
Do we still need this change, or can it be reverted? Looks like it's not actually being used.